### PR TITLE
Features/Multilingual Language Switcher #141275029

### PR DESCRIPTION
**Pull request by kaptron**
still WIP for now, just creating PR

### CODE REVIEW

File: spec/javascripts/components/short_form_language_switcher_spec.coffee [hunk 1]
```diff
@@ -0,0 +1,27 @@
+do ->
```
Comment (chrisdolendo) [inline spec/javascripts/components/short_form_language_switcher_spec.coffee]: @kaptron do you notice any weird syntax errors here? I just noticed when running the tests that this file wasn't picked up.

**Comment by chrisdolendo** [comment]
story delivered, ready for engineering review.

File: spec/javascripts/components/short_form_language_switcher_spec.coffee (hunk 1 above)
Comment (kaptron) [inline spec/javascripts/components/short_form_language_switcher_spec.coffee]: @chrisdolendo seems like you got this sorted out? looks like the spec is running

**Comment by kaptron** [comment]
👍

**Pull request merged**